### ISSUE TITLE
2017-02-07

### ISSUE THREAD

**Issue by xaviervia** [batata-frita/meta#1]

**Comment by xaviervia** [comment]
- Nobody seems to be shipping unbuild babel projects to npm, so we should not prio that in Sagui.

**Comment by xaviervia** [comment]
- Generating content (screencasts & blog posts) around the projects would be interesting.

**Comment by xaviervia** [comment]
- Sagui can probably de-prio CSS modules

**Comment by xaviervia** [comment]
- Wouldn’t it be nice to have a _snooze_ button on github issues?
- NPM or some other package manager could implement donations?

**Comment by xaviervia** [comment]
- An idea, probably not happening: move react-entanglement to a monorepo in batata-frita?

- It’s currently fairly hard to test different adapters
- We need to figure out a way to follow up on React Entanglement

**Comment by xaviervia** [comment]
Electron in Sagui will be done only after it's tried out in other projects: https://github.com/saguijs/sagui/pull/308#issuecomment-278099929

**Comment by xaviervia** [comment]
To follow up each project we probably want to do in a separate day: 
- https://github.com/macaco-maluco/day-one needs to merge the big refactor branch

**Comment by xaviervia** [comment]
Prios:

1. Move mature projects to batata-frita (react-context-props, git-revision-webpack-plugin are low hanging fruit)
2. Sagui Webpack 2: first step though is to verify that the approach for extract text webpack plugin is not a hack
3. Merge Day One refactor
4. React Component for open source landing pages, probably using material ui or something like that. The idea is to be able to generate landing pages so that there is less friction around publishing. Once this is done(/while it's being done), the idea is to use it for React Entanglement.
5.  Put aside some time to submit mature projects to curated lists
